packages/test-helper-server/package.json

```diff
@@ -18,7 +18,8 @@
     "express": "^4.13.3",
     "mkdirp": "^0.5.1",
     "morgan": "^1.6.1",
-    "multer": "^1.2.0"
+    "multer": "^1.2.0",
+    "uuid": "^3.0.1"
```

Comment:
right. I'm pretty sure they didn't cause problems because of npm3 making them available (or possibly because they were installed in the root node_modules directory)